python/tvm/contrib/cc.py

```diff
@@ -212,9 +214,9 @@ def _linux_compile(output, objects, options, compile_cmd="g++", compile_shared=F
         raise RuntimeError(msg)
 
 
-def _windows_shared(output, objects, options):
+def _windows_compile(output, objects, options):
     cmd = ["clang"]
-    cmd += ["-O2", "-flto=full", "-fuse-ld=lld-link"]
+    cmd += ["-O2", "-v", "-fvisibility=default", "-export-all-symbols"]
```

Comment:
Can we check if we need `-export-all-symbols` after https://github.com/apache/tvm/pull/6698? I believe we should have marked the life scopes correctly.

Reply:
Yes, there are some debug flags in there too, -v. That line needs to be cleaned up.

Reply:
OK, thanks, please add a comment about the need of debug flags there.

Reply:
The current code in the PR only has the flag `-O2`. The other flags are not required.